RELEASE-NOTES.md

```diff
@@ -1,23 +1,65 @@
 # Release Notes
 
-This is the `john-packages` (itself) 0.9 release 'v0.9.0'. October 2023 (version 0.9.0)
+## Release [rolling-2404](https://github.com/openwall/john-packages/releases/tag/rolling-2404) (2024-04-05)
+
+Version `1.9J1+2404`. This is a maintenance release:
 
-## This is [rolling-2310](https://github.com/openwall/john-packages/releases/tag/rolling-2310) (2023-10-05)
+April’s bugfix release.
 
 ### Breaking Changes
 
+- Due to hardware unavailability, we have stopped building the macOS X86 (Intel CPU) package;
+- Due to hardware limitations, we have stopped building SSE2 binaries (only on Linux for now).
+
+### Bugfixes
+
+- Some random bugfixes here and there.
+
+### Improvements
+
+- Add arm64 Docker image (with NVIDIA GPU support);
+- Add new formats and/or implementations;
+  - Armory wallet, Keplr wallet, Argon2 OpenCL.
+- *2john improvements;
+  - Add a modern version of pdf2john (.py; Python 3).
+- Improve OpenCL detection;
+  - Start using dynamic OpenCL binding (dynamically load OpenCL library);
+  - Add Khronos Group OpenCL header files (use this local copy of the OpenCL headers);
+- Some random optimizations here and there;
+- Minor documentations tweaks.
+
+### Binaries Available
+
+* Docker (for X86_64 and aarch64) with NVIDIA GPU support (via OpenCL);
+* Flatpak;
+  * bundle (for X86_64);
+  * via FlatHub (for X86_64 and aarch64).
+* macOS (for M1 and above);
+* Snap (for X86_64, armhf, arm64, ppc64el, riscv64, and s390x);
+* Windows (for X86_64).
+
+All except s390x and riscv64 binaries support SIMD.
+
+------
+
+## Release [rolling-2310](https://github.com/openwall/john-packages/releases/tag/rolling-2310) (2023-10-05)
+
+This is the `john-packages` (itself) 0.9 release 'v0.9.0'. October 2023 (version 0.9.0)
+
+### Breaking Changes (2310)
+
 - Due to changes in Cygwin, plus OS and hardware unavailability in common cloud build environments,
   we deprecate 32 bits architectures.
   - For now, Canonical snap packages for armhf can still be created.
 - Intel-based macOS resources are sunsetting everywhere.
   - Probably the rolling-2310 will be the last release for this architecture.
 
-### Bugfixes
+### Bugfixes (2310)
```

Comment:
This change is necessary due to complaints from linters. I found out that "linters" were fixed upstream, we'll stop doing things in the future.